vsdownload.py

```diff
@@ -303,6 +303,17 @@ def matchPackageHostArch(p, host):
         if "host" + a in id:
             return a == host
 
+    # Here we apply the knownledge that
+    # - x64 host can run x86 binaries
+    # - arm64 host can run x86/x64 binaries
```

Comment:
Hmm - I don't think we in general can assume that arm64 hosts can run x86 or x64 binaries? That's generally not the case, or what do you have in mind?

(In particular, even if you'd be running in an environment where you'd have something like Rosetta or qemu set up to emulate x86 binaries on arm64, you'd still need to have a suitable wine setup for the foreign architecture - which I wouldn't assume is the case in general.)

Reply:
> I don't think we in general can assume that arm64 hosts can run x86 or x64 binaries? That's generally not the case

It is the case for Windows. So it it possible that in the manifest, some arm64 packages depend on x64 packages and depend on x86 packages.

Reply:
Right, I see what you mean.